### ISSUE TITLE
Auto-select first available task not working if list is empty

### ISSUE THREAD

**Issue by ManuelDittmar** [camunda/zeebe#18687]
### Describe the bug

[auto-select.webm](https://github.com/camunda/zeebe/assets/83572388/ea8cc9bc-621c-42b7-be72-fb43c2796d69)

### To Reproduce

1. Start Instance
2. Complete Task A
3. Wait for Task B

--> Task B is not selected.

### Actual behavior

Next available task is not displayed.

### Expected behavior

Task B is displayed.

### Environment

- Browser: [Chrome]
- Tasklist Version: [e.g. 8.5.1]

### Additional context

---------------------------------------------------------------------------------------------

### Definition of Ready - Checklist

- [ ] The bug has been reproduced by the assignee in the environment compatible with the provided one; otherwise, the issue is closed with a comment
- [ ] The issue has a meaningful title, description, and testable acceptance criteria
- [ ] Necessary screenshots, screen recordings, or files are attached to the bug report

For UI changes required to solve the bug:

- [ ] Design input has been collected by the assignee

### After solving the issue - Answer and document in the comments the questions below

- [ ] What was the root cause for this bug?
- [ ] Is there anything we should do for the error to not happen anymore?

**Comment by douglasbouttell-camunda** [comment]
Reviewing the video it seems like auto select is working as expected.

When completing the task it reloads the tasklist to select the next task. Since there are no tasks it goes to the home page. 

For some reason the home page does not render properly.

There will be a delay until zeebe creates the next task so we may not always be able to navigate to the next task in a process immediately.

**Comment by ManuelDittmar** [comment]
@douglasbouttell-camunda, yes it reloads the tasks, and you can see on the left side that also the task appears, however i would expect the task to be selected and not only listed on the left. I am not expecting it to show the next task immediately, but i would expect that it's auto-selected, once it exists.
